### ISSUE TITLE
deprecated `validate_bool()` function

### ISSUE THREAD

**Issue by chadh** [sensu/puppet-module-sensuclassic#29]
## Description of problem

I noticed warnings like this in the puppetserver logs:
```
2020-06-01T16:13:11.870Z WARN  [qtp605080165-47] [puppetserver] Puppet This method is deprecated, please use the stdlib validate_legacy
 function,
                    with Stdlib::Compat::Bool. There is further documentation for validate_legacy function in the README. at ["/etc/pup
petlabs/code/environments/production/modules/sensuclassic/manifests/init.pp", 631]:["/etc/puppetlabs/code/environments/production/site-
modules/role/manifests/common.pp", 8]
...
```

**Comment by chadh** [comment]
PR incoming

**Comment by ghoneycutt** [comment]
Released in v3.5.1
